<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowForm</class>
 <widget class="QMainWindow" name="MainWindowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>411</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="w">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,3,1">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkMode">
          <property name="text">
           <string>Solid</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderOpacity">
          <property name="value">
           <number>99</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkPocket">
          <property name="text">
           <string>Pockets</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,7,3">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QListWidget" name="listId"/>
        </item>
        <item>
         <widget class="chemkit::GraphicsView" name="view" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tbl">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="console">
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="maximumBlockCount">
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenPdb"/>
   <addaction name="actionGetPdb"/>
   <addaction name="separator"/>
   <addaction name="actionConsole"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionFirst"/>
   <addaction name="actionBack"/>
   <addaction name="actionNext"/>
   <addaction name="actionLast"/>
   <addaction name="separator"/>
   <addaction name="actionShowPDB"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionOpenPdb">
   <property name="text">
    <string>openPdb</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>actionAbout</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionGetPdb">
   <property name="text">
    <string>getPdb</string>
   </property>
   <property name="toolTip">
    <string>Download from PDB</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>showConsole</string>
   </property>
   <property name="toolTip">
    <string>Show console</string>
   </property>
  </action>
  <action name="actionFirst">
   <property name="text">
    <string>First</string>
   </property>
   <property name="toolTip">
    <string>First</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>Back</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next</string>
   </property>
  </action>
  <action name="actionLast">
   <property name="text">
    <string>Last</string>
   </property>
   <property name="toolTip">
    <string>Last</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete All</string>
   </property>
  </action>
  <action name="actionShowPDB">
   <property name="text">
    <string>T</string>
   </property>
   <property name="toolTip">
    <string>Show text</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>chemkit::GraphicsView</class>
   <extends>QWidget</extends>
   <header location="global">chemkit/graphicsview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
